pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -94,12 +90,8 @@ in {
   parallel = doJailbreak super.parallel;
   path = doJailbreak super.path;
   polyparse = overrideCabal (drv: { postPatch = "sed -i -e 's, <0.11, <0.12,' polyparse.cabal"; }) (doJailbreak super.polyparse);
-  primitive = dontCheck (doJailbreak self.primitive_0_7_4_0);
   regex-posix = doJailbreak super.regex-posix;
-  resolv = doJailbreak super.resolv;
   singleton-bool = doJailbreak super.singleton-bool;
```

Comment:
`stackage.yaml` has v0.1.6 which allows base-4.17 (GHC 9.4). So this shouldn't be needed.